app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -240,26 +244,36 @@ def build_metric_id(type, resource, metric_id)
     # selected target server. This server is identified
     # by ems_ref, which in Hawkular terms is the
     # fully qualified resource path from Hawkular inventory
-    def run_generic_operation(operation, ems_ref, parameters = {})
-      with_provider_connection do |connection|
+    def run_generic_operation(dmr_operation, ems_ref, parameters = {})
         the_operation = {
-          :operationName => operation,
+          :operationName => dmr_operation,
```

Comment:
It is clear when you come from an AS7/WildFly background. It stands for something like 'detyped model representation'.

But we should change it back to `operation_name`, as the input is indeed no DMR-operation.